202/tests/ProductSerializer/TestAttributeTestCase.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Copyright since 2019 Shopping Feed
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to tech@202-ecommerce.com so we can send you a copy immediately.
+ *
+ * @author    202 ecommerce <tech@202-ecommerce.com>
+ * @copyright Since 2019 Shopping Feed
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
+ */
+
+namespace Tests\OrderImport;
+
+use PHPUnit\Framework\TestCase;
+use ShoppingfeedAddon\Actions\ActionsHandler;
+use ShoppingfeedPreloading;
+use ShoppingfeedToken;
+use Tools;
+
+class TestAttributeTestCase extends TestCase
+{
+    public function testGetProductAvailabilityLabelWithStockAndAvailableMessage(): void
+    {
+        $id_product = 1;
+        $token = (new ShoppingfeedToken())->getDefaultToken();
```

Comment:
You can set id_shoppingfeed_token = 1 because data fixtures are reset by run_for_unittest.sh script